backend/src/main/java/com/notbadcode/todo/task/TaskResource.java

```diff
@@ -0,0 +1,151 @@
+package com.notbadcode.todo.task;
+
+import com.notbadcode.todo.task.dto.TaskDto;
+import com.notbadcode.todo.task.dto.TasksInfoDto;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.PATCH;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@Path("/tasks")
+public class TaskResource {
+  private final TaskService taskService;
+  private final HttpServletRequest request;
+  private final Logger log;
+
+  @Autowired
+  public TaskResource(TaskServiceImpl taskService,
+                      HttpServletRequest request) {
+    this.taskService = taskService;
+    this.request = request;
+    this.log = LoggerFactory.getLogger(TaskResource.class);
+  }
+
+  @GET
+  @Path("/{id}")
+  public Response getTaskById(@PathParam("id") Long id) {
+    logUrlInfo();
+    TaskDto task = taskService.findTaskById(id);
+    return Response
+        .status(Response.Status.OK)
+        .entity(task)
+        .type(MediaType.APPLICATION_JSON)
+        .build();
+  }
```

Comment:
Где в ответе 200 с какой-то дтошкой, можно спокойно возвращать саму дтошку.
```suggestion
  @GET
  @Path("/{id}")
  @Produces(MediaType.APPLICATION_JSON)
  public TaskDto getTaskById(@PathParam("id") Long id) {
    logUrlInfo();
    return taskService.findTaskById(id);
  }
```